src/main/java/org/broadinstitute/consent/http/service/EmailNotifierService.java

```diff
@@ -217,17 +207,6 @@ public void sendAdminFlaggedDarApproved(String darCode, List<User> admins, Map<U
         }
     }
 
-    public void sendUserDelegateResponsibilitiesMessage(User user, Integer oldUser, String newRole, List<Vote> delegatedVotes) throws MessagingException, IOException, TemplateException {
```

Comment:
Nice ... this is such an outdated method - we removed delegation at least a year ago.